packages/core/src/SearchInput/SearchInput.tsx

```diff
@@ -0,0 +1,42 @@
+import { forwardRef } from "react";
+import {
+  ExtractNames,
+  useDefaultProps,
+} from "@hitachivantara/uikit-react-utils";
+
+import { HvAdornment } from "../FormElement";
+import { HvIcon } from "../icons";
+import { HvInput, HvInputProps } from "../Input";
+import { staticClasses, useClasses } from "./SearchInput.styles";
+
+export { staticClasses as searchInputClasses };
+
+export type HvSearchInputClasses = ExtractNames<typeof useClasses>;
+
+export const HvSearchInput = forwardRef<
+  React.ComponentRef<"input">,
+  HvInputProps<HTMLInputElement>
+>(function HvSearchInput(props, ref) {
+  const { classes: classesProp, ...others } = useDefaultProps(
+    "HvSearchInput",
+    props,
+  );
+
+  const { classes } = useClasses(classesProp);
+
+  return (
+    <HvInput
+      ref={ref}
+      type="search"
+      disableSearchButton
+      startAdornment={
+        <HvAdornment icon={<HvIcon compact name="Search" title={"Search"} />} />
```

Comment:
this isn't/doesn't need to be a11y visible, so we don't need the label. and that's good, because we shouldn't hard-code it otherwise
```suggestion
        <HvAdornment icon={<HvIcon compact name="Search" />} />
```